lms/djangoapps/courseware/views/views.py

```diff
@@ -261,6 +261,7 @@ def jump_to(_request, course_id, location):
     return redirect(redirect_url)
 
 
+@login_required
```

Comment:
What user will see in this case?

Reply:
@TamoshaytisV answered, please see my comment below. Thanks